money-core-scala/src/main/scala/com/comcast/money/core/CoreSpanFactory.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2012-2015 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.comcast.money.core
+
+import com.comcast.money.api.{ Span, SpanFactory, SpanHandler, SpanId }
+
+import scala.collection.JavaConversions._
+
+class CoreSpanFactory(handler: SpanHandler) extends SpanFactory {
+
+  def newSpan(spanName: String): Span = newSpan(new SpanId(), spanName)
+
+  def childSpan(childName: String, span: Span): Span = childSpan(childName, span, false)
+
+  def childSpan(childName: String, span: Span, sticky: Boolean): Span = {
+    val child = newSpan(span.info.id.newChildId, childName)
+
+    if (sticky) {
+      span.info.notes.values
+        .filter(_.isSticky)
+        .foreach(child.record)
+    }
```

Comment:
@clinedome-work yes, we added this, it is an existing feature today.  In CAM and XfinityAPI, Wally wanted to see things like RecordingID and AccountNumber on ALL Spans, so that he can do some interesting queries.  Doing things like Transactions in SPLUNK wound up being a PITA.

So, when we record, we have the option to mark a note as "Sticky", in which case all Stick notes automatically propagate to child spans.